a3p-integration/proposals/b:restart-vats/submission/restart-vats-permit.json

```diff
@@ -0,0 +1,25 @@
+{
+  "consume": {
+    "contractKits": true,
+    "governedContractKits": true,
+    "instancePrivateArgs": true,
+    "loadCriticalVat": true,
+    "psmKit": true,
+    "vatStore": true,
+    "zoe": "zoe",
+    "provisioning": "provisioning",
+    "vaultFactoryKit": true,
+    "agoricNamesAdmin": "makeCoreProposalBehavior",
+    "vatAdminSvc": "makeCoreProposalBehavior"
+  },
+  "produce": {},
+  "evaluateBundleCap": "makeCoreProposalBehavior",
+  "installation": {
+    "produce": "makeCoreProposalBehavior"
+  },
+  "modules": {
+    "utils": {
+      "runModuleBehaviors": "makeCoreProposalBehavior"
+    }
+  }
+}
```

Comment:
github flags lack of a newline at end of file here.
Do we have a shop norm?
Locally, I get a lot of spurious diffs on save for this sort of thing.

Reply:
We don't have `.editorconfig` set up to specify line endings but I think the norm is to include them.

Reply:
This is just a file move of the output from `agoric run`. If we want the file here to have a newline, I think the proper fix is to update upstream. (I don't think it's worth it)